src/scss/includes/search_form.scss

```diff
@@ -351,11 +358,16 @@ input[type="search"] {
     }
 
     .search_form__logo {
-      width: 26px;
-      height: 26px;
-      background-size: 26px;
-      top: 15px;
-      left: 16px;
+      display: none;
+    }
+
+    .main_field_return {
+      display: block;
+      left: 10px;
+      top: 20px;
+      font-size: 25px;
+      color: #353C52;
```

Comment:
Can you use the `$primary_text` variable here? It'll make it easier to keep things consistent during the redesign.
(and I'll probably make a PR to generalize the use of color vars instead of hard-coded value).